src/passkey_child/passkey_child.c

```diff
@@ -71,6 +72,13 @@ int main(int argc, const char *argv[])
         }
     } else if (data.action == ACTION_AUTHENTICATE) {
         ERROR("This action isn't implemented yet.\n");
+    } else if (data.action == ACTION_FORMAT_PEM) {
+        ret = format_pem_public_key(main_ctx, &data, &pem_key);
+        if (ret != EOK) {
+            ERROR("Error formatting key.\n");
```

Comment:
Print error code?

Reply:
I don't think it makes sense to print an error code outside of its original error message. This error line is used in a general way to indicate that the formatting has failed, if the user wants to understand why, then they will need to read the extended debugging/error messages.

Reply:
ok